src/Arcus.Scripting.ApiManagement/Scripts/Set-AzApiManagementApiSubscriptionKey.ps1

```diff
@@ -0,0 +1,14 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroup,
+    [Parameter(Mandatory = $true)][string] $ServiceName,
+    [Parameter(Mandatory = $true)][string] $ApiId,
+    [Parameter(Mandatory = $false)][string] $ApiKeyHeaderName = "x-api-key",
+    [Parameter(Mandatory = $false)][string] $ApiQueryParamName = "apiKey"
+)
+
+$apimContext = New-AzApiManagementContext -ResourceGroupName $ResourceGroup -ServiceName $ServiceName
+Write-Host "Using API Management instance '$ServiceName' in resource group '$ResourceGroup'"
+
+Set-AzApiManagementApi -Context $apimContext -ApiId $ApiId -Protocols "https" -SubscriptionKeyHeaderName $ApiKeyHeaderName -SubscriptionKeyQueryParamName $ApiQueryParamName
```

Comment:
Can we leave out protocols? Because it shouldn't be our job to change those.

If not, I would do a get first and pass the same protocols.

Reply:
This parameter is optional indeed, so we shouldn't include this, as we're purely talking about the subscription-key.

Reply:
Removed this. Resolving.

Reply:
If it's not included, does it keep the same values @mbraekman?